webpack.config.js

```diff
@@ -109,6 +109,7 @@ module.exports = (env = {}) => {
         // needed to reference Vuetify styles in the shadow DOM
         vuetify: path.resolve('node_modules', 'vuetify'),
         static: staticFilesDir,
+        '^vuetify/src/stylus$': false,
```

Comment:
This regex looked like it would never match anything because it's too specific (the `^` and `$` mean it would need to be an exact full line match). I deleted it, and everything still worked as before, so I don't think we need to add this.